test/extended/dr/common.go

```diff
@@ -337,6 +336,40 @@ func ssh(cmd string, node *corev1.Node) (*e2essh.Result, error) {
 	return e2essh.IssueSSHCommandWithResult(cmd, e2e.TestContext.Provider, node)
 }
 
+// deleteEtcdPodsOnNodes will find etcd static pods running on the nodes and remove their mirror pods from the apiserver.
+// This is used to clean left-over state, which has been stored in a snapshot and does not accurately reflect reality anymore.
+func deleteEtcdPodsOnNodes(client kubernetes.Interface, nodes []*corev1.Node) error {
```

Comment:
this is not needed anymore since the OVN steps are added, because kubelet was missing apiserver connection to update the status.

I'll leave this here for now, in case we may need it for another test. We can do a cleanup run after the feature branch closes.